nucypher/characters/lawful.py

```diff
@@ -539,7 +544,12 @@ def threshold_decrypt(self,
 
         # Now, the decryption share can be used to decrypt the ciphertext
         shared_secret = combine_decryption_shares(shares)
-        cleartext = decrypt_with_shared_secret(ciphertext, conditions, shared_secret, generator)
+        cleartext = decrypt_with_shared_secret(
+            ciphertext,
+            json.dumps(conditions).encode(),  # TODO: ferveo core needs to support Conditions
```

Comment:
What does it need to do specifically?

Reply:
I was considering using `nucypher_core.Conditions` but I'm having second thoughts about this comment actually, I think the `aad` field is appropriate as `bytes`.

Reply:
Yes, we were considering AAD to be used as a "transport" for conditions.